### PR TITLE
Docs for team sidebar

### PR DESCRIPTION
**Pull request by jasonblais**

### CODE REVIEW

File: source/help/getting-started/creating-teams.rst [hunk 1]
```diff
@@ -6,11 +6,17 @@ New teams can be created if the System Admin has **Enable Team Creation** set to
...
 
-Team Selection Page
+Team Sidebar
 ~~~~~~~~~~~~~~~~~~~~
-Navigate to the web address of your system, ``https://domain.com/``. Sign in using your credentials. Click **Create a new team** to be guided through the rest of the set up steps. If this option is not visible on the web page, then the System Admin has disabled team creation. 
+If you belong to more than one team, a team sidebar will appear left of your channel list on the left-hand sidebar.
+
+.. image:: ../images/team-sidebar.png
+
+Click the "+" icon at the bottom of the team sidebar, then click **Create a new team** on the team selection page, which guides you through the rest of the set up steps. If this option is not visible on the web page, then the System Admin has disabled team creation.
```
Comment (JeffSchering) [inline source/help/getting-started/creating-teams.rst]: Use set up as a verb, and setup as a noun or adjective. So in this case, it should be setup.

File: source/help/getting-started/signing-in.rst [hunk 2]
```diff
@@ -4,8 +4,9 @@ Signing in
 --------------
 
 You can sign in to your team from the root page of your Mattermost site
-``https://domain.com``. After signing in, you will be able to view a
-list of your teams and select which one to open.
+``https://domain.com``. After signing in, the team that appears first on your
```
Comment (JeffSchering) [inline source/help/getting-started/signing-in.rst]: You should use example.com, which is a special domain set aside for use in documentation. Domain.com is an actual, existing, website.

Also, the way it's worded here, it sounds like you sign in to Mattermost by going to https://domain.com

Comment (jasonblais) [inline source/help/getting-started/signing-in.rst]: Should it be "You can sign in to your team from the root page of your Mattermost site, ``https://<domain>.com``" instead?

There, `<domain>` would be inserted by the user,

Comment (JeffSchering) [inline source/help/getting-started/signing-in.rst]: Thinking about it some more, if you do something like http://<domain>.com, users still need to know what to replace <domain> with, and they get that from the administrator or by some other means. So they pretty much already know where the sign-in page is, and if they don't, then the doc should help them with that. So maybe  you could say something like:

*To sign in, navigate to the Mattermost sign-in page. You can get the URL of the sign-in page from your administrator. After you sign in, you can view a list of your teams and select which one to open.*

File: source/help/getting-started/configuring-notifications.rst [hunk 3]
```diff
@@ -92,11 +92,22 @@ being viewed on desktop.
    from **Account Settings** > **Notifications** > **Mobile Push
    Notifications** > **Trigger push notifications when**.
 
+Team Sidebar Notifications
+----------------------------------------
+
+If Mattermost is open in a browser tab the favicon updates to inform you of unread messages and
```
Comment (esethna) [inline source/help/getting-started/configuring-notifications.rst]: This needs an updated intro paragraph?

File: source/help/getting-started/configuring-notifications.rst [hunk 4]
```diff
@@ -92,11 +92,22 @@ being viewed on desktop.
...
+mentions. Browser tab notifications are available on Firefox and Chrome.
+
+- Unread messages are denoted by a small dot left of the team icon.
+- Unread mentions are displayed with a mention counter similar to channels and appears on the top right corner of the team icon.
+
+.. image:: ../images/team-sidebar-notifications.png
```
Comment (esethna) [inline source/help/getting-started/configuring-notifications.rst]: Image is broken. Might need to be `../../images/team-sidebar-notifications.png`

File: source/help/getting-started/configuring-notifications.rst [hunk 5]
```diff
@@ -92,11 +92,22 @@ being viewed on desktop.
...
    Notifications** > **Trigger push notifications when**.
 
+Team Sidebar Notifications
+----------------------------------------
+
+If Mattermost is open in a browser tab the favicon updates to inform you of unread messages and
+mentions. Browser tab notifications are available on Firefox and Chrome.
+
+- Unread messages are denoted by a small dot left of the team icon.
+- Unread mentions are displayed with a mention counter similar to channels and appears on the top right corner of the team icon.
```
Comment (esethna) [inline source/help/getting-started/configuring-notifications.rst]: Propose: `Unread mentions are displayed with a mention counter that appears on the top right corner of the team icon.`

File: source/help/getting-started/creating-teams.rst [hunk 6]
```diff
@@ -6,11 +6,17 @@ New teams can be created if the System Admin has **Enable Team Creation** set to
...
-Teams can be created from the Team Selection page or Main Menu. The person who creates a team will automatically be given the Team Admin role for that team.
+Teams can be created from the team sidebar or Main Menu. The person who creates a team will automatically be given the Team Admin role for that team.
 
-Team Selection Page
+Team Sidebar
 ~~~~~~~~~~~~~~~~~~~~
-Navigate to the web address of your system, ``https://domain.com/``. Sign in using your credentials. Click **Create a new team** to be guided through the rest of the set up steps. If this option is not visible on the web page, then the System Admin has disabled team creation. 
+If you belong to more than one team, a team sidebar will appear left of your channel list on the left-hand sidebar.
+
+.. image:: ../images/team-sidebar.png
```
Comment (esethna) [inline source/help/getting-started/creating-teams.rst]: Image broken. See comment above

File: source/help/getting-started/creating-teams.rst [hunk 7]
```diff
@@ -6,11 +6,17 @@ New teams can be created if the System Admin has **Enable Team Creation** set to
 Methods to Create a Team
 ------------------------
 
-Teams can be created from the Team Selection page or Main Menu. The person who creates a team will automatically be given the Team Admin role for that team.
+Teams can be created from the team sidebar or Main Menu. The person who creates a team will automatically be given the Team Admin role for that team.
```
Comment (esethna) [inline source/help/getting-started/creating-teams.rst]: Maybe this should start with, `If team creation is enabled by the System Admin, ...`

File: source/help/getting-started/signing-in.rst [hunk 8]
```diff
@@ -66,9 +67,10 @@ Mattermost.
 Switching Teams
 ---------------
 
-You can switch between teams you have joined using the **Main Menu**,
-accessed by clicking the three dots in the top header of any team site
-on the server.
+You can switch between teams you have joined using the team sidebar 
+that appears left of your channel list on the left-hand sidebar.
+
+.. image:: ../images/team-sidebar.png
```
Comment (esethna) [inline source/help/getting-started/signing-in.rst]: image broken, see comment above

File: source/help/settings/team-settings.md [hunk 9]
```diff
@@ -19,7 +17,9 @@ You can enter a name up to 15 characters in length. The length of team names is
 
 #### Allow anyone to join this team
 
-After a user logs in to the site, they are shown a Team Selection page. Any team with **Allow anyone to join this team** option set to **Yes** will show up on this page under "Open teams you can join".
+After a user logs in to the site, they are shown a Team Selection page if they don't belong to a team. Any team with **Allow anyone to join this team** option set to **Yes** will show up on this page under "Teams you can join". 
+
+A user can access the Team Selection page by also clicking the "+" icon at the bottom of their team sidebar or from the **Main Menu** > **Join Another Team**.
```
Comment (esethna) [inline source/help/settings/team-settings.md]: Join another team is on mobile only, maybe we should state that?

File: source/help/getting-started/configuring-notifications.rst (hunk 3 above)
Comment (jasonblais) [inline source/help/getting-started/configuring-notifications.rst]: err yeah..thanks for the catch

File: source/help/settings/team-settings.md (hunk 9 above)
Comment (jasonblais) [inline source/help/settings/team-settings.md]: It's also on desktop

**Comment by jasonblais** [comment]
Feedback addressed

File: source/help/settings/team-settings.md (hunk 9 above)
Comment (esethna) [inline source/help/settings/team-settings.md]: Oh true, thanks! I was looking on a team with no public teams left to join.

**Comment by lfbrock** [comment]
It looks good, the images look a little fuzzy though (especially the one only showing the one team with notification jewel) - is it possible to make them more clear?

**Comment by jasonblais** [comment]
@lfbrock how are the new ones?

**Comment by lfbrock** [comment]
Yeah I think that's fine, thanks jason. +1

**Pull request merged**